.circleci/config.yml

```diff
@@ -97,6 +97,11 @@ jobs:
     environment:
       DOCKER_IMAGE: "308535385114.dkr.ecr.us-east-1.amazonaws.com/caffe2/py2-clang8-ubuntu16.04:283"
     <<: *linux_default
+  PYTORCH:
+    resource_class: 2xlarge+
```

Comment:
xlarge? I do not see this in the documentation: https://circleci.com/docs/2.0/configuration-reference/#resource_class

Reply:
hmmmm this is a recommendation of tvm. will double check.